components/automate-cli/cmd/chef-automate/gather-logs.go

```diff
@@ -452,6 +452,12 @@ func runGatherLogsLocalCmd(outfileOverride string, logLines uint64) error {
 	g.AddCopiesFromPath("config", "/hab/svc")
 	g.AddCopiesFromPath("logs", "/hab/svc")
 
+	// A2-chef-server-related files
```

Comment:
Am i missing something here:
```
[272][default:/src:0]# ls /hab/svc/automate-cs-bookshelf/etc
ls: cannot access '/hab/svc/automate-cs-bookshelf/etc': No such file or directory
[273][default:/src:2]# ls /hab/svc/automate-cs-nginx/etc
ls: cannot access '/hab/svc/automate-cs-nginx/etc': No such file or directory
[275][default:/src:0]# ls /hab/svc/automate-cs-oc-bifrost/etc
ls: cannot access '/hab/svc/automate-cs-oc-bifrost/etc': No such file or directory
[276][default:/src:2]# ls /hab/svc/automate-cs-oc-bifrost/etc
ls: cannot access '/hab/svc/automate-cs-oc-bifrost/etc': No such file or directory
```

Reply:
`AddCopiesFromPath` isn't a join, so the path can be in any subdirectory under `/hab/svc`; in this case, `/etc/` is mostly under `/hab/svc/<service>/var/etc`:

```
[25][default:/src:126]# ls /hab/svc/automate-cs-bookshelf/var/
etc  logs
[26][default:/src:0]# ls /hab/svc/automate-cs-nginx/var/
logs  stdout  tmp
[27][default:/src:0]# ls /hab/svc/automate-cs-oc-bifrost/var/
etc  logs
[28][default:/src:0]# ls /hab/svc/automate-cs-oc-erchef/var/
etc  logs
```